CHANGELOG.asciidoc

```diff
@@ -80,6 +80,7 @@ https://github.com/elastic/beats/compare/v5.3.0...master[Check the HEAD diff]
 *Packetbeat*
 
 *Winlogbeat*
+- Deprecated metrics endpoint. Is replaced in 6.0 by http endpoint.
```

Comment:
How about something like this that clues them on where to look for the replacement. "It is superseded by a libbeat feature that can serve metrics on an HTTP endpoint."